### PR TITLE
[DOCS-1653] Zipkin tracing plugin bump version, add missing configs (2.4 CE beta/GA), add datatypes

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCS-1653 part of epic https://konghq.atlassian.net/browse/DOCS-1396.

Schema link:

https://github.com/Kong/kong-plugin-zipkin/blob/master/kong/plugins/zipkin/schema.lua

Direct review link:

https://deploy-preview-2713--kongdocs.netlify.app/hub/kong-inc/zipkin/ 

Doc reviewers: focus mainly on the index.md file for review. The older versions had some feedback applied as requested by the SME.

Reviewers: @gszr and @kikito I noticed that this plugin was recently updated to 1.3.x, so I bumped the version while I'm adding datatypes. I noticed some fields in the schema not in the docs, and some fields in the docs not in the schema. I noted that in the index file. I'll need new descriptions as I requested directly in the source. If you were planning to update the plugin docs, please feel free to work directly in this PR.

(Update) I just saw the announcement for 2.4 beta https://kongstrong.slack.com/archives/C0DS8F44W/p1616543679040800, and this plugin was mentioned in the changelog:

https://github.com/Kong/kong/blob/2.4.0-beta.1/CHANGELOG.md#plugins

### CODE REVIEW

File: app/_hub/kong-inc/zipkin/index.md [hunk 1]
```diff
@@ -60,51 +61,66 @@ params:
...
         The `b3-single` option expects or adds Zipkin's B3 single-header tracing headers.
         The `w3c` option expects or adds W3C's traceparent tracing header. The `preserve` option
-        does not expect any format, and will  transmit whatever header is recognized or present,
-        defaulting to `b3` if none is found. In case of mismatch between the expected and incoming
+        does not expect any format, and will transmit whatever header is recognized or present,
+        with a default of `b3` if none is found. In case of a mismatch between the expected and incoming
         tracing headers (for example, when `header_type` is set to `b3` but a w3c-style tracing header is
         found in the incoming request), then the plugin will add both kinds of tracing headers
-        to the request and generate a mismatch warning in the logs.
+        to the request and generate a mismatch warning in the logs. XX REVIEWERS need a description for jaeger and ot.
```
Comment (kikito) [inline app/_hub/kong-inc/zipkin/index.md:103]: `jaeger
```suggestion
        to the request and generate a mismatch warning in the logs. `jaeger` will use and expect [Jaeger-style tracing headers](https://www.jaegertracing.io/docs/1.22/client-libraries/#propagation-format) (`uber-trace-id`). The `ot` option is for [OpenTelemetry tracing headers](https://github.com/open-telemetry/opentelemetry-java/blob/96e8523544f04c305da5382854eee06218599075/extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/OtTracerPropagator.java) of the form `ot-tracer-*`.
```

File: app/_hub/kong-inc/zipkin/index.md [hunk 2]
```diff
@@ -60,51 +61,66 @@ params:
...
+      datatype: string
+      description: |
+        XX REVIEWERS need a description
     - name: static_tags
       required: false
       default: {}
       value_in_examples: { { name = "color", value = "red" } }
+      datatype: array
       description: |
         The tags specified on this property will be added to the generated request traces.
```
Comment (kikito) [inline app/_hub/kong-inc/zipkin/index.md:125]:
```suggestion
    - name: static_tags
      required: false
      default: []
      value_in_examples: [ { name = "color", value = "red" } ]
      description: |
        The tags specified on this property will be added to the generated request traces.
```

**Review by kikito — CHANGES_REQUESTED** [review]

File: app/_hub/kong-inc/zipkin/1.2.x.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      description: |
+        Allows specifying the type of header to be added to requests with no pre-existing tracing headers
+        and when `config.header_type` is set to `"preserve"`.
+        When `header_type` is set to any other value, `default_header_type` is ignored.
+    - name: static_tags
+      required: false
+      default: {}
+      value_in_examples: { { name = "color", value = "red" } }
+      description: |
+        The tags specified on this property will be added to the generated request traces.
```
[the remark this answers is not in the document]
Comment (tlblessing) [reply app/_hub/kong-inc/zipkin/1.2.x.md:109]: Okay thank you for clarifying. This  was already like that, not anything I added.  Will adjust all versions.

File: app/_hub/kong-inc/zipkin/index.md [hunk 4]
```diff
@@ -60,51 +61,66 @@ params:
...
         How often to sample requests that do not contain trace ids.
-        Set to `0` to turn sampling off, or to `1` to sample **all** requests.
+        Set to `0` to turn sampling off, or to `1` to sample **all** requests. The
+        value must be between zero (0) and one (1), inclusive.
     - name: default_service_name
       required: false
+      datatype: string
       description: |
-        The default service name to override the unknown-service-name spans.
+        The default service name to override the unknown-service-name spans. XX REVIEWERS this option does not appear in the schema?
```
[the remark this answers is not in the document]
Comment (tlblessing) [reply app/_hub/kong-inc/zipkin/index.md]: done

File: app/_hub/kong-inc/zipkin/index.md [hunk 5]
```diff
@@ -60,51 +63,62 @@ params:
...
         and when `config.header_type` is set to `"preserve"`.
-        When `header_type` is set to any other value, `default_header_type` is ignored.
+        When `header_type` is set to any other value, `default_header_type` is ignored. Possible values are
+        `b3`, `b3-single`, `w3c`, `jaeger`, or `ot`.
+    - name: tags_header
+      required: true
+      default: Zipkin-Tags
+      datatype: string
+      description: |
+        XX REVIEWERS need a description
```
Comment (tlblessing) [inline app/_hub/kong-inc/zipkin/index.md:117]: @kikito need description

Comment (kikito) [inline app/_hub/kong-inc/zipkin/index.md:117]: The Zipkin plugin will add extra headers to the tags associated with any HTTP requests that come with a header named as configured by this property. The format is `name_of_tag=value_of_tag`, separated by commas. For example: with the default value, a request with the header `Zipkin-Tags: fg=blue, bg=red` will generate a trace with the tag `fg` with value `blue`, and another tag called `bg` with value `red`.

**Review by kikito — CHANGES_REQUESTED** [review]
Missed a description, added now

**Review by kikito — APPROVED** [review]

**Review by GayleNeumann — APPROVED** [review]
Looks great!

**Pull request merged**